ui/ui-components/components/record/SampleRecordCard.tsx

```diff
@@ -135,7 +137,11 @@ const SampleRecordCard: React.FC<SampleRecordCardProps> = ({
   );
 
   const saveRecord = useCallback(
-    (record, groups = undefined, destinations = undefined) => {
+    (
+      record: RecordType,
+      groups: Models.GroupType[],
+      destinations: Models.DestinationType[]
```

Comment:
Since these consistently get values now it always displays the Groups and Destinations sections, but will correctly show `None`